### DISCUSSION TITLE
how can i get the elements fit inside?

### DISCUSSION THREAD

**Discussion by semi710** [ErikReider/SwayNotificationCenter#416] [Q&A]
![image](https://github.com/ErikReider/SwayNotificationCenter/assets/60490474/53a77b3a-db1e-4cfc-bc46-9494ec5e763b)

What css element i need to modify to get these buttons fit in the notification?

**Comment by semi710** [comment]
@ErikReider if you can help!

**Reply by ErikReider** [reply]
Are you using the default CSS?

**Reply by semi710** [reply]
Nop it was from config flex ..... I changed some padding that caused this now fixed.
